config/cashier_plans.php

```diff
@@ -60,7 +61,13 @@
             /**
              * The length of the billing cycle.
              */
-            'interval' => '1 month',
+            // 'interval' => 1 month or
+            'interval' => [
+                'generator' => AdvancedIntervalGenerator::class,
+                'value' => 1,
+                'period' => 'month', /* day, month or year*/
+                'fixed' => false, /*  If fixed is true the interval is Carbon Base Interval Generator use NoOverflow, */
```

Comment:
Do you have a suggestion for a variable name that is more verbose, more clear about what it does?